src/core_tests.rs

```diff
@@ -38,6 +38,9 @@ use std::sync::mpsc;
 use std::thread;
 use types::MessageId;
 use xor_name::XorName;
+use maidsafe_utilities;
+// Add the following line to tests to enable logging:
```

Comment:
Maybe you could create a snippet or alias for this then? (I use a snippet personally)  I don't see why they should be included in this file and not every other source file.